temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivityPollTask.java

```diff
@@ -67,6 +68,7 @@ public Boolean apply(LocalActivityWorker.Task task, Duration maxWaitAllowed) {
       }
       return accepted;
     } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
```

Comment:
Here previously we potentially were completely loosing an interrupted flag of a worker thread. [WorkflowWorker -> ReplayWorkflowTaskHandler -> ReplayWorkflowRunTaskHandler -> LocalActivityPollTask#apply]